src/app/components/table_details/table_details.js

```diff
@@ -94,10 +108,17 @@ angular
                     relation = database + model.schema + "." + model.alias;
                 }
 
+                var owner_identifier
+                if (model.group) {
+                    owner_identifier = retrieveOwner(`group.${model.package_name}.${model.group}`)
+                } else {
+                    owner_identifier = metadata.owner
+                }
```

Comment:
For an if/else you can use the [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) during assignment.
```suggestion
                const owner_identifier = model.group
                    ? retrieveOwner(`group.${model.package_name}.${model.group}`)
                    : metadata.owner;
```